cc3200-sys/src/socket.rs

```diff
@@ -0,0 +1,713 @@
+    ESEC_UNSUPPORTED_CERTIFICATE = -343, // ssl/tls alerts
+    ESEC_CERTIFICATE_REVOKED = -344, // ssl/tls alerts
+    ESEC_CERTIFICATE_EXPIRED = -345, // ssl/tls alerts
+    ESEC_CERTIFICATE_UNKNOWN = -346, // ssl/tls alerts
+    ESEC_ILLEGAL_PARAMETER = -347, // ssl/tls alerts
+    ESEC_UNKNOWN_CA = -348, // ssl/tls alerts
+    ESEC_ACCESS_DENIED = -349, // ssl/tls alerts
+    ESEC_DECODE_ERROR = -350, // ssl/tls alerts
+    ESEC_DECRYPT_ERROR = -351, // ssl/tls alerts
+    ESEC_EXPORT_RESTRICTION = -360, // ssl/tls alerts
+    ESEC_PROTOCOL_VERSION = -370, // ssl/tls alerts
+    ESEC_INSUFFICIENT_SECURITY = -371, // ssl/tls alerts
+    ESEC_INTERNAL_ERROR = -380, // ssl/tls alerts
+    ESEC_USER_CANCELLED = -390, // ssl/tls alerts
+    ESEC_NO_RENEGOTIATION = -400, // ssl/tls alerts
+    ESEC_UNSUPPORTED_EXTENSION = -410, // ssl/tls alerts
+    ESEC_CERTIFICATE_UNOBTAINABLE = -411, // ssl/tls alerts
+    ESEC_UNRECOGNIZED_NAME = -412, // ssl/tls alerts
+    ESEC_BAD_CERTIFICATE_STATUS_RESPONSE = -413, // ssl/tls alerts
+    ESEC_BAD_CERTIFICATE_HASH_VALUE = -414, // ssl/tls alerts
+    // propierty secure
+    ESECGENERAL = -450, // error secure level general error
+    ESECDECRYPT = -451, // error secure level, decrypt recv packet fail
+    ESECCLOSED = -452, // secure layrer is closed by other size , tcp is still connected
+    ESECSNOVERIFY = -453, // Connected without server verification
+    ESECNOCAFILE = -454, // error secure level CA file not found
+    ESECMEMORY = -455, // error secure level No memory  space available
+    ESECBADCAFILE = -456, // error secure level bad CA file
+    ESECBADCERTFILE = -457, // error secure level bad Certificate file
+    ESECBADPRIVATEFILE = -458, // error secure level bad private file
+    ESECBADDHFILE = -459, // error secure level bad DH file
+    ESECT00MANYSSLOPENED = -460, // MAX SSL Sockets are opened
+    ESECDATEERROR = -461, // connected with certificate date verification error
+    ESECHANDSHAKETIMEDOUT = -462, // connection timed out due to handshake time
+}
+
+pub type RawSocket = i16;
+
+// Casts a RawSocket into an Error to check the return value of sl_Socket.
+impl ::core::convert::Into<SocketError> for RawSocket {
+    fn into(self) -> SocketError {
+        unsafe { ::core::intrinsics::transmute::<RawSocket, SocketError>(self) }
+    }
+}
+
+// Used for return types that are either an error or a buffer size.
+pub type SizeOrError = i16;
+
+// Used to convert a SizeOf Error into a Result for functions like sl_Send
+impl ::core::convert::TryInto<SocketError> for SizeOrError {
+    type Err = ();
+    fn try_into(self) -> Result<SocketError, Self::Err> {
+        if self >= 0 {
+            return  Err(());
+        }
+        Ok(unsafe { ::core::intrinsics::transmute::<SizeOrError, SocketError>(self) })
+    }
+}
+
+// Enum for the return type of SL_FD_ISSET
+#[repr(i16)]
+#[allow(non_camel_case_types)]
+pub enum SlBoolean {
+    True = 1,
+    Fals = 0,
```

Comment:
Is that a typo? Or is the misspelling deliberate? If deliberate, it's probably worth a comment.